nemesis-generator/configuration_generator/__main__.py

```diff
@@ -11,16 +11,18 @@
 
 
 class NemesisConfigurationGenerator:
-	def __init__(self, network_name, network_identifier):
-		self.facade = NemFacade(network_name)
-		self.network_identifier = int(network_identifier) if network_identifier else self.facade.network.identifier
-		self.network = Network(network_name, self.network_identifier, self.facade.network.datetime_converter.epoch)
+	def __init__(self, network_name, custom_network_identifier):
+		network_identifier = (
+			int(custom_network_identifier) if custom_network_identifier else Network.MAINNET.identifier if 'mainnet' == network_name else
+			Network.TESTNET.identifier
+		)
+		self.facade = NemFacade(Network(network_name, network_identifier, Network.MAINNET.datetime_converter.epoch))
```

Comment:
why are you using `Network.MAINNET.datetime_converter` ?

Reply:
mainnet and testnet have the same value so didnt think it matters but I will add a check.

Reply:
ah, that's right ... it's NEM 😅